src/game/default/g_entity.c

```diff
@@ -71,6 +133,9 @@ static g_entity_spawn_t g_entity_spawns[] = { // entity class names -> spawn fun
 
 	{ "worldspawn", G_worldspawn },
 
+	// compatibility-only entities
+	{ "weapon_chaingun", G_weapon_chaingun },
+
```

Comment:
Why isn't it adequate to simply map `weapon_chaingun` to `G_weapon_lightning` ? We'll never have a map that contains both. Then the G_weapon_chaingun functions above just go away.